### PR TITLE
Readme file changes for adding teams channel in bot

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**
Added steps for enabling teams channel in bot registration.

### CODE REVIEW

File: samples/app-auth/nodejs/README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ This sample demonstrates authentication in Microsoft Teams apps.
     ```
 
 1. Register a bot with Azure Bot Service, following the instructions [here](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-quickstart-registration?view=azure-bot-service-3.0).
+- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/en-us/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
 1. While registering the bot, use `https://<your_ngrok_url>/api/messages` as the messaging endpoint.
```
Comment (v-chetsh) [inline samples/app-auth/nodejs/README.md:33]: Why this is added as a point 1. ?
Also I can see that the next point is also starting with 1. 
image

File: samples/bot-request-approval/nodejs/Readme.md [hunk 2]
```diff
@@ -61,7 +61,8 @@ This sample shows a feature where:
 ### 1. Setup for Bot
 In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2).
 
-### 2. Run your bot sample
+- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/en-us/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)### 2. Run your bot sample
```
Comment (v-chetsh) [inline samples/bot-request-approval/nodejs/Readme.md:64]: Is this correct here?
"### 2. Run your bot sample"

image

**Review by v-chetsh — CHANGES_REQUESTED** [review]
Reviewed and provided my feedback.

**Review by v-chetsh — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**